cluster/examples/kubernetes/ceph/toolbox-job.yaml

```diff
@@ -0,0 +1,68 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: rook-ceph-toolbox-job-script
+  namespace: rook-ceph
+data:
+  script: |
+    #!/bin/bash
+    # Modify this script to run any ceph, rbd, radosgw-admin, or other commands that could
+    # be run in the toolbox pod. The output of the commands can be seen by getting the pod log.
+    ceph status
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: rook-ceph-toolbox-job
+  namespace: rook-ceph
+  labels:
+    app: ceph-toolbox-job
+spec:
+  template:
+    spec:
+      initContainers:
+      - name: config-init
+        image: rook/ceph:master
```

Comment:
I don't think we need the operator image for this. We should simply point to a Ceph container image `ceph/ceph`.